paths_cli/tests/commands/test_contents.py

```diff
@@ -72,6 +74,7 @@ def test_contents_table(tps_fixture, table):
             ],
         }[table]
         assert results.output.split("\n") == expected
+        assert_click_success(results)
         assert results.exit_code == 0
```

Comment:
`assert_click_succes` already asserts the exit code
```suggestion
        assert_click_success(results)
```